includes/footer.php

```diff
@@ -1,21 +1,91 @@
-        <div class="body-container footer" style="">
+<div class="body-container footer" style="" xmlns:background-image="http://www.w3.org/1999/xhtml">
```

Comment:
Sorry, that was an import I made and later removed the css that used it but forgot to remove the import itself